docs/source/configuration/environmentvariables.md

```diff
@@ -9,193 +9,186 @@ myst:
+    ```
 
-### VOLTO_ROBOTSTXT
+`VOLTO_ROBOTSTXT`
+    You can override the {file}`robots.txt` file with an environment variable called `VOLTO_ROBOTSTXT`.
+    This is useful when using the same build on multiple websites (for example a test website) to forbid robots from crawling it.
+    The following example is a two-line shell command.
 
-You can override the robots.txt file with an environment variable called
-`VOLTO_ROBOTSTXT`. This is useful when using the same build on multiple
-websites (for example a test website) to forbid robots from crawling it.
+    ```shell
+    VOLTO_ROBOTSTXT="User-agent: *
+    Disallow: /" yarn start
+    ```
 
-```
-$ VOLTO_ROBOTSTXT="User-agent: *
-Disallow: /" yarn start
-```
+    ```{note}
+    If you want to use the `VOLTO_ROBOTSTXT` environment variable, your must delete the file `public/robots.txt` from your project to avoid a conflict.
+    ```
 
-```{note}
-If you want to use the `VOLTO_ROBOTSTXT` environment variable, make sure to
-delete the file `public/robots.txt` from your project.
-```
+`DEBUG`
+    It enables several logging points scattered throughout the Volto code.
+    It uses the `volto:` namespace.
+    You can use it in a specific named scope or everywhere.
 
-### DEBUG
+    `shadowing` enables component shadowing errors.
+    It displays the errors of the non-compliant customizations in the server console.
+    It helps you identify problems with a customization that does not work as you expect.
 
-It will enable the log several logging points scattered through the Volto code. It uses the `volto:` namespace.
+    ```shell
+    DEBUG=volto:shadowing yarn start
+    ```
 
-```bash
-DEBUG=volto:i18n yarn start
-```
+    `i18n` enables the log of missing internationalization messages in the console.
 
-or
+    ```shell
+    DEBUG=volto:i18n yarn start
+    ```
 
-```bash
-DEBUG=volto:shadowing yarn start
-```
+    `*` enables logging everywhere it exists in Volto.
 
-also
+    ```shell
+    DEBUG=volto:* yarn start
+    ```
 
-```bash
-DEBUG=volto:* yarn start
-```
+`DEBUG_ADDONS_LOADER`
+    Set `DEBUG_ADDONS_LOADER=true` to have Volto generate a file, {file}`addon-dependency-graph.dot`, which contains a graph of all the loaded add-ons.
+    You can use [Graphviz](https://graphviz.org/) to convert this file to an image with:
 
-#### `DEBUG_ADDONS_LOADER`
+    ```
+    dot addon-dependency-graph.dot -Tsvg -o out.svg
+    ```
 
-Set `DEBUG_ADDONS_LOADER=true` to have Volto generate a file, `addon-dependency-graph.dot` which contains a graph of all the loaded addons. You can use [Graphviz](https://graphviz.org/) to convert this file to an image with:
+`ADDONS`
+    You can use the `ADDONS` environment variable to enable and configure add-ons in your project.
 
-```
-dot addon-dependency-graph.dot -Tsvg -o out.svg
-```
+    When running your app, the add-ons will be loaded in the following order:
 
-#### Component shadowing errors (shadowing)
+    -   the file `package.json`
```

Comment:
I think it's worth to say that it's additive, never removes or invalidates the others.

Reply:
Done in [72f294f](https://github.com/plone/volto/pull/5736/commits/72f294fc54be7c786bd71bb15e0fc70713f8ae02)